src/ts-node.ts

```diff
@@ -75,6 +75,7 @@ function registerTSNode(root: string) {
         sourceMap: true,
         rootDirs,
         typeRoots,
+        jsx: 'react',
```

Comment:
`jsx: tsconfig.compilerOptions.jsx || 'react'` Is anyone going to want to use `preserve` mode?

https://www.typescriptlang.org/docs/handbook/jsx.html#basic-usage

Reply:
`preserve` mode would typically be used if
- Your JS environment has native support for JSX syntax
- You're going to pass it through another transpile step

Since `oclif` is managing the transpilation here, I don't think it's something that we have to worry about. Anyone who would currently be using JSX with Oclif would have had to put their own transpilation process in front of it anyways, and so would be unaffected.